gce/scylla_create_devices

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+#
+# Copyright 2020 ScyllaDB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import sys
+from pathlib import Path
+from subprocess import run, PIPE
+
+
+def get_nvme_devices():
+    result = run(["ls /dev/nvme0n*"], stdout=PIPE, text=True, shell=True)
+    devices = result.stdout.strip().split()
+    assert devices, "No NVMe devices found!"
+    return devices
+
+
+def create_raid(devices):
+    scylla_path = Path("/var/lib/scylla")
+    if scylla_path.is_mount():
+        print(f"{scylla_path} is already mounted. Will not run 'scylla_raid_setup'!")
+        sys.exit(0)
+    run(["/opt/scylladb/scripts/scylla_raid_setup", "--raiddev", "/dev/md0", "--disks", ",".join(devices),
```

Comment:
Can you parametrize --raiddev and --root? Simillar to https://github.com/scylladb/scylla-machine-image/pull/50/files#diff-6e97616a1e8944b01fcc6ef68257e0d6
This script will be used by the other PR once both of them will get merged.